src/Microsoft.AspNet.Server.Kestrel/KestrelEngine.cs

```diff
@@ -1,26 +1,31 @@
 // Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-using System;
 using Microsoft.AspNet.Server.Kestrel.Networking;
-using System.Collections.Generic;
-using System.Threading.Tasks;
 using Microsoft.AspNet.Server.Kestrel.Http;
 using Microsoft.Framework.Runtime;
+using Microsoft.Framework.Logging;
+using Microsoft.Framework.Logging.Console;
+using System;
+using System.Collections.Generic;
 using System.IO;
+using System.Threading.Tasks;
 
 namespace Microsoft.AspNet.Server.Kestrel
 {
     public class KestrelEngine : IDisposable
     {
 
-        public KestrelEngine(ILibraryManager libraryManager)
+        public KestrelEngine(ILibraryManager libraryManager, ILoggerFactory loggerFactory)
         {
             Threads = new List<KestrelThread>();
             Listeners = new List<Listener>();
             Memory = new MemoryPool();
             Libuv = new Libuv();
 
+            loggerFactory.AddConsole();
```

Comment:
@lodejard before I merge this, do you have any thoughts on this? Is it OK for Kestrel to register the Console logger? Presumably an app might register the Console logger as well and if it does I think it'll print every message twice (once from each console logger). Maybe that's not really a problem because apps have to ultimately be aware of which hosts they're running on, so they have to know that under Kestrel to not add the console logger.

Thoughts?